src/chains/tests/test_views.py

```diff
@@ -174,10 +171,9 @@ def test_match(self):
                 "text_color": chain.theme_text_color,
                 "background_color": chain.theme_background_color,
             },
-            "gas_price_oracle": {
-                "url": chain.gas_price_oracle_url,
-                "gas_parameter": chain.gas_price_oracle_parameter,
-                "gwei_factor": "{0:.9f}".format(chain.gas_price_oracle_gwei_factor),
+            "gas_price": {
+                "type": "fixed",
```

Comment:
Does this imply this endpoint will be polymorphic?

Reply:
Yes right now only two types available: `fixed` and `oracle`